### ISSUE TITLE
It seems that record_modifier is not included in image 3.1.0

### ISSUE THREAD

**Issue by OlivierJavaux** [fluent/fluent-bit#9068]
I have upgraded fluentbit in my kubernetes cluster with version 3.1.0.
But I have an error at startup: [config] section 'record_modifier' tried to instance a plugin name that doesn't exist
It seems that record_modifier plugin is no more included in fluentbit 3.1.0.

```
$ docker run -it --rm cr.fluentbit.io/fluent/fluent-bit:3.1.0 -R | grep record_modifier
$
```
It was included in 3.0.0 release:
```
$ docker run -it --rm cr.fluentbit.io/fluent/fluent-bit:3.0.0 -R | grep record_modifier
  record_modifier         modify record
```

**Comment by patrick-stephens** [comment]
Yes, this is fixed in 3.1.1

**Comment by OlivierJavaux** [comment]
Fixed in 3.1.1. Thanks @patrick-stephens

**Comment by sepek** [comment]
FTR: It's not in the registry yet
